modules/base/shaders/modelOpacity_fs.glsl

```diff
@@ -0,0 +1,64 @@
+/*****************************************************************************************
+ *                                                                                       *
+ * OpenSpace                                                                             *
+ *                                                                                       *
+ * Copyright (c) 2014-2022                                                               *
+ *                                                                                       *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of this  *
+ * software and associated documentation files (the "Software"), to deal in the Software *
+ * without restriction, including without limitation the rights to use, copy, modify,    *
+ * merge, publish, distribute, sublicense, and/or sell copies of the Software, and to    *
+ * permit persons to whom the Software is furnished to do so, subject to the following   *
+ * conditions:                                                                           *
+ *                                                                                       *
+ * The above copyright notice and this permission notice shall be included in all copies *
+ * or substantial portions of the Software.                                              *
+ *                                                                                       *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,   *
+ * INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A         *
+ * PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT    *
+ * HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF  *
+ * CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE  *
+ * OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                                         *
+ ****************************************************************************************/
+
+#include "fragment.glsl"
+#include "floatoperations.glsl"
+
+in vec2 vs_st;
+
+uniform float opacity = 1.0;
+
+uniform sampler2D colorTexture;
+uniform sampler2D depthTexture;
+
+uniform ivec4 viewport;
+uniform vec2 resolution;
+
+Fragment getFragment() {
+  Fragment frag;
+
+  // Modify the texCoord based on the Viewport and Resolution. This modification is
+  // necessary in case of side-by-side stereo as we only want to access the part of the
+  // feeding texture that we are currently responsible for.  Otherwise we would map the
+  // entire feeding texture into our half of the result texture, leading to a doubling of
+  // the "missing" half.  If you don't believe me, load a configuration file with the
```

Comment:
Maybe consider another phrasing than "If you don't believe me"? :)

Reply:
This is copied from an already existing comment in the [FXAA](https://github.com/OpenSpace/OpenSpace/blob/14f0ad7338371107e81a2e9af2c32bf332408407/shaders/framebuffer/fxaa.frag#L51), [HDR](https://github.com/OpenSpace/OpenSpace/blob/14f0ad7338371107e81a2e9af2c32bf332408407/shaders/framebuffer/hdrAndFiltering.frag#L48), [volume shader](https://github.com/OpenSpace/OpenSpace/blob/14f0ad7338371107e81a2e9af2c32bf332408407/shaders/framebuffer/mergeDownscaledVolume.frag#L36), and maybe some other places too where this calculation is also done. I kept it for consistency but I could change it too.